src/Pages/HomePage/componants/ActivityItem .tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+function ActivityItem(props: { name: string; backgroundColor: string; time: string; amount :string;icon:string}) {
```

Comment:
add a seperate interface Props{} to define the type and use JS Destructing objects to access them 

```
interface Props{...}
function ActivityItem( { name, backgroundColor, time, amount,icon}:Props)
```